src/types/types.ts

```diff
@@ -100,3 +100,9 @@ export interface iLine {
   sublines?: iSubline[];
   sugiaName?: string;
 }
+
+export interface iManuscriptPopup{
```

Comment:
Here, maybe it will be useful also to set the entire relevant iSubline.
(so we can display the text, for example, and the related synopsis)